recipe/meta.yaml

```
@@ -1,47 +1,62 @@
 {% set name = "routingpy" %}
-{% set version = "0.3.3" %}
-
+{% set version = "1.3.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/routingpy-{{ version }}.tar.gz
-  sha256: 297425fccf8911aa19e73ae245c5cb6464f4674eeceb9f699891bcb5f9d1d089
+  # Starting from v0.3.3 the source code isn't available on PyPI anymore (only wheels)
+  # To make sure that the git repo is available we should use the git_url and git_tag
+  # because without them setuptools_scm won't find the git tag and a version will be 'None'.
+  git_url: https://github.com/gis-ops/{{ name }}.git
+  git_tag: {{ version }}
 
 build:
   number: 0
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
+  build:
+    - git  # [not win]
   host:
     - pip
-    - python >=3.6
-    - poetry >=0.12
+    - python
+    - setuptools
+    - setuptools_scm
+    - wheel
   run:
-    - python >=3.6
-    - requests >=2.0.0,<3.0.0
+    - python
+    - requests >=2.20.0,<3.0.0
   run_constrained:
-    - pandas >=1.1.0,<2
-    - folium >=0.11.0,<1
-    - shapely >=1.7.0,<2
-    - ipykernel >=5.3.4,<6
+    - matplotlib-base >=3.4.1,<4
+    - contextily >=1.1.0,<1.2.0
+    - geopandas >=0.8.2,<1
```

```suggestion
    - geopandas >=0.8.2,<0.9.0
```
https://python-poetry.org/docs/dependency-specification/#caret-requirements